message_definitions/v1.0/development.xml

```diff
@@ -14,6 +14,15 @@
         <description>True if the data from this sensor is being actively used by the flight controller for guidance, navigation or control.</description>
       </entry>
     </enum>
+    <enum name="RADIO_RC_CHANNELS_FLAGS" bitmask="true">
+      <description>RADIO_RC_CHANNELS flags (bitmask).</description>
+      <entry value="1" name="RADIO_RC_CHANNELS_FLAGS_FAILSAFE">
+        <description>Failsafe is active.</description>
+      </entry>
+      <entry value="2" name="RADIO_RC_CHANNELS_FLAGS_FRAME_MISSED">
```

Comment:
this can't be filled in accurately without the sender knowing if any mavlink frames were lost, as we can't know if this is new data or not.
Much better to have an 8 bit number of how many 10s of milliseconds since we received the last valid message from the receiver

Reply:
could also be a 16bit or 32 bit millisecond timestamp in the senders time domain

Reply:
decided on call we will use a 32 bit ms timestamp since boot on the senders time domain, and remove the FRAME_MISSED flag

Reply:
THX
will be done